src/components/BookmarkButton/BookmarkButtonSmall.tsx

```diff
@@ -45,7 +44,7 @@ export const HeartIcon = styled(Icon).attrs(
   ({ bookmarked }: HeartIconProps) => ({
     fill: "#F42D97",
     stroke: "none",
-    name: bookmarked && "heartBookmarkFilled",
+    name: bookmarked ? "heartBookmarkFilled" : "", // pink dot appears if `bookmarked && ...`
```

Comment:
This will still leave the icon svg in the end code, just as an empty tag === bad because empty components are clutter. Better to move the conditional rendering inside the return function.